config/examples/redhat_customer_portal.yml

```diff
@@ -1,19 +1,15 @@
 base: &default
-  enabled: true
-  client_id:
-  client_secret:
-  realm:
-  flow:
-  entitlements_notifications_enabled: true
+  enabled: <%= ENV.fetch('RH_CUSTOMER_PORTAL_ENABLED', false) %>
+  client_id: <%= ENV.fetch('RH_CUSTOMER_PORTAL_CLIENT_ID', '3scale') %>
+  client_secret: <%= ENV['RH_CUSTOMER_PORTAL_CLIENT_SECRET'] %>
+  realm: <%= ENV.fetch('RH_CUSTOMER_PORTAL_REALM', 'implicit') %>
+  flow: <%= ENV.fetch('RH_CUSTOMER_PORTAL_FLOW', 'implicit') %>
+  entitlements_notifications_enabled: <%= ENV.fetch('RH_SUPPORT_ENTITLEMENTS_NOTIFICATIONS', '1') == '1' %>
   assign_entitlements_email: <%= ENV.fetch('RH_ASSIGN_ENTITLEMENTS_EMAIL', 'assign.entitlements@example.com') %>
-  revoke_entitlements_email: <%= ENV.fetch('RH_ASSIGN_ENTITLEMENTS_EMAIL', 'revoke.entitlements@example.com') %>
+  revoke_entitlements_email: <%= ENV.fetch('RH_REVOKE_ENTITLEMENTS_EMAIL', 'revoke.entitlements@example.com') %>
```

Comment:
probably idea was to set only one env variable and use for both. If you want to add ability to change separately, I htink we should still use `RH_ASSIGN_ENTITLEMENTS_EMAIL` for revoke if provided, unless `RH_REVOKE_ENTITLEMENTS_EMAIL` is also present.

Reply:
I think that's quite confusing to use `RH_ASSIGN_ENTITLEMENTS_EMAIL` for `revoke_entitlements_email` property :upside_down_face: 

In the [only place](https://github.com/3scale/3scale-saas/blob/master/manifests/3scale-saas/system/overlays/pro-saas/resources/secrets/config/redhat_customer_portal.yml#L7-L8) where this integration is used the emails are separate (although surprisingly, they are not under env var).

I'd keep it as it is - I think it's not a big deal to set another env var, even if it would be the same value.

Reply:
If `RH_ASSIGN_ENTITLEMENTS_EMAIL` is not used in production, then alright.